### PR TITLE
Update CI

### PR DESCRIPTION
**Pull request by lu-zero**
In preparation of the avx512 support.
- Bump nightly version
- Bump minimum version

### CODE REVIEW

File: .github/workflows/rav1e.yml [hunk 1]
```diff
@@ -268,9 +268,9 @@ jobs:
         CARGO_INCREMENTAL: 0
         RUSTFLAGS: >
           -Zprofile -Ccodegen-units=1 -Clink-dead-code -Coverflow-checks=off
-          -Zno-landing-pads
+          -Cpanic=abort -Zpanic_abort_tests
```
Comment (tdaede) [inline .github/workflows/rav1e.yml:271]: Is this equivalent to -Zno-landing-pads?

Comment (lu-zero) [reply .github/workflows/rav1e.yml:271]: Yes

Comment (lu-zero) [reply .github/workflows/rav1e.yml:271]: Yes

Comment (Luni-4) [reply .github/workflows/rav1e.yml:271]: Add also `RUSTDOCFLAGS: -Cpanic=abort` to avoid problems with doc-tests

**Comment by Luni-4** [comment]
@lu-zero 

While at it, update `grcov` to `0.5.15`, thanks!

File: .github/workflows/rav1e.yml (hunk 1 above)
Comment (lu-zero) [reply .github/workflows/rav1e.yml:271]: Sure :)

**Comment by lu-zero** [comment]
@tdaede you need to remove the check from the settings.

**Comment by shssoichiro** [comment]
I got it.

**Comment by shssoichiro** [comment]
I'm going to go ahead and merge this, since with the settings update, 1.43.1 run is now required to merge anything. It looks like tdaede's concerns should have been resolved.

**Pull request merged**